tests/dwn.spec.ts

```diff
@@ -26,14 +28,16 @@ export function testDwnClass(): void {
     // important to follow the `before` and `after` pattern to initialize and clean the stores in tests
     // so that different test suites can reuse the same backend store for testing
     before(async () => {
+      didResolver = new DidResolver({ didResolvers: [DidKey] });
+
       const stores = TestStores.get();
       messageStore = stores.messageStore;
       dataStore = stores.dataStore;
       eventLog = stores.eventLog;
 
       eventStream = TestEventStream.get();
 
-      dwn = await Dwn.create({ messageStore, dataStore, eventLog, eventStream });
+      dwn = await Dwn.create({ didResolver, messageStore, dataStore, eventLog, eventStream });
```

Comment:
What's the need for doing this?

Reply:
When working on this feature I linked this repo to `dwn-sql-store` to run the test suite and make sure I didn't run into any gotchas w/ the tag indexing.

I started getting failures that didn't seem related to what I was adding, and when debugging I was getting a DB locked error from the resolver (default `Dwn.create()` uses the LevelDB cache for the resolver).

I did want to spend a little time digging into what changed between the previous release and now, but haven't gotten a chance to. However, this behavior is consistant when I link latest `main` in both repos.

Reply:
Got it, so if I understand correctly, this little change worked around the DB lock issue when test suite is running in `dwn-sql-store` repo.

Feel free to close if I understood correctly.